pandora-common/src/networking/directory_client.ts

```diff
@@ -101,6 +101,40 @@ export const ACCOUNT_SETTINGS_DEFAULT = Object.freeze<IDirectoryAccountSettings>
 	interfaceChatroomOfflineCharacterFilter: 'ghost',
 });
 
+export const DirectoryAccountSettingsLimitedSchema = z.object({
+	displayName: DisplayNameSchema,
+});
+export type IDirectoryAccountSettingsLimited = z.infer<typeof DirectoryAccountSettingsLimitedSchema>;
```

Comment:
The `I` prefix is inconsistent throughout the codebase, so I'm slowly trying to phase it out. I think it is better without it:
```suggestion
export type DirectoryAccountSettingsLimited = z.infer<typeof DirectoryAccountSettingsLimitedSchema>;
```